### PR TITLE
Basic Subdomain takeover finding uplift

### PR DESCRIPTION
**Pull request by RRudder**

### CODE REVIEW

**Review by vortexau — CHANGES_REQUESTED** [review]

File: submissions/description/server_security_misconfiguration/misconfigured_dns/basic_subdomain_takeover/template.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Basic subdomain takeover could lead to data theft and indirect financial loss through the attacker’s ability to interact with legitimate users. These malicious actions could also result in reputational damage for the business through the impact to customers’ trust.
+
+## Steps to Reproduce
+
+1. Register DNS record using a registration service and point to the following endpoint: {{URL}}
+1. Browse to the URL `takeover.inscope.com/proof.txt`
+1. You will see a Bugcrowd Username, for example:
+
+```text
```
[the remark this answers is not in the document]
Comment (RRudder) [reply submissions/description/server_security_misconfiguration/misconfigured_dns/basic_subdomain_takeover/template.md]: @vortexau please see latest commit addressing these changes

File: submissions/description/server_security_misconfiguration/misconfigured_dns/basic_subdomain_takeover/template.md [hunk 2]
```diff
@@ -14,11 +14,7 @@ Basic subdomain takeover could lead to data theft and indirect financial loss th
 
 1. Register DNS record using a registration service and point to the following endpoint: {{URL}}
```
Comment (evildaemond) [inline submissions/description/server_security_misconfiguration/misconfigured_dns/basic_subdomain_takeover/template.md:15]: Quick change to remove this as this is not required for repro

**Review by evildaemond — CHANGES_REQUESTED** [review]
1 comment then LGTM

File: submissions/description/server_security_misconfiguration/misconfigured_dns/basic_subdomain_takeover/template.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (evildaemond) [reply submissions/description/server_security_misconfiguration/misconfigured_dns/basic_subdomain_takeover/template.md]: Fixed in latest version

File: submissions/description/server_security_misconfiguration/misconfigured_dns/basic_subdomain_takeover/template.md (hunk 2 above)
Comment (evildaemond) [reply submissions/description/server_security_misconfiguration/misconfigured_dns/basic_subdomain_takeover/template.md]: Fixed

**Review by evildaemond — APPROVED** [review]

**Pull request merged**